### PR TITLE
add query string to static assets with cms version

### PR DESCRIPTION
**Pull request by vxsx**
in order to bust cache every time new release is made

### CODE REVIEW

**Comment by mkoistinen** [comment]
LGTM

File: cms/templates/admin/cms/page/change_form.html [hunk 1]
```diff
@@ -16,11 +16,11 @@
 {{ block.super }}
 {# load nonconflict pre and post files to fix conflict if there is another jQuery loaded #}
 {# the code is included by a external file because the code has to be loaded into a python file where the wizard is loaded as well #}
-<script src="{% static "cms/js/modules/jquery.noconflict.pre.js" %}" type="text/javascript"></script>
-<script src="{% static "admin/js/urlify.js" %}" type="text/javascript"></script>
-<script src="{% static "cms/js/dist/bundle.admin.base.min.js" %}" type="text/javascript"></script>
-<script src="{% static "cms/js/dist/bundle.admin.changeform.min.js" %}" type="text/javascript"></script>
-<script src="{% static "cms/js/modules/jquery.noconflict.post.js" %}" type="text/javascript"></script>
+<script src="{% static "cms/js/modules/jquery.noconflict.pre.js" %}?{% get_cms_version %}" type="text/javascript"></script>
```
Comment (czpython) [inline cms/templates/admin/cms/page/change_form.html]: this should be a template tag that wraps ``{% static %}`` maybe called ``static_with_version``.

File: cms/templatetags/cms_admin.py [hunk 2]
```diff
@@ -252,6 +253,9 @@ def admin_static_url():
     """
     return getattr(settings, 'ADMIN_MEDIA_PREFIX', None) or ''.join([settings.STATIC_URL, 'admin/'])
 
+@register.simple_tag
+def get_cms_version():
+    return __version__
```
Comment (czpython) [inline cms/templatetags/cms_admin.py]: here you should define the ``static_with_version`` templatetag

File: cms/forms/wizards.py [hunk 3]
```diff
@@ -77,11 +78,11 @@ class PageTypeSelect(forms.widgets.Select):
     """
     class Media:
         js = (
-            'cms/js/modules/jquery.noconflict.pre.js',
-            'cms/js/dist/bundle.admin.base.min.js',
-            'cms/js/modules/cms.base.js',
-            'cms/js/widgets/wizard.pagetypeselect.js',
-            'cms/js/modules/jquery.noconflict.post.js'
+            'cms/js/modules/jquery.noconflict.pre.js?%s' % __version__,
```
Comment (czpython) [inline cms/forms/wizards.py]: maybe reuse the logic from ``static_with_version``?

**Comment by czpython** [comment]
:-1: from me. We can talk and see how to improve it though, too much repetition at the moment.

**Pull request closed without merging**